etc/signal-desktop.profile

```diff
@@ -0,0 +1,35 @@
+# Firejail profile for sinal-desktop
+# This file is overwritten after every install/update
+# Persistent local customizations
+include /etc/firejail/signal-desktop.local
+# Persistent global definitions
+include /etc/firejail/globals.local
+
+noblacklist ~/.config/Signal
+
+include /etc/firejail/disable-common.inc
+include /etc/firejail/disable-devel.inc
+include /etc/firejail/disable-programs.inc
```

Comment:
What about `include /etc/firejail/disable-passwdmgr.inc`?

Reply:
Yes, we can include this